RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,157 @@
+
+    // Serial execution, so all launch parameters set to 1.
+#if defined(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED)
+    adjustedBlocks = Vec::all(static_cast<Idx>(1));
+    adjustedThreads = Vec::all(static_cast<Idx>(1));
+#endif
+
+    // Threads enabled, set number of blocks to 1.
+#if defined(ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLED)
+    adjustedBlocks = Vec::all(static_cast<Idx>(1));
+#endif
+
+    return WorkDiv(adjustedBlocks, adjustedThreads, elementsPerThreadArg);
+  }
+
+// If a compile time flag does not define PT_CUT, default to 0.8 (GeV)
+#ifndef PT_CUT
+  constexpr float PT_CUT = 0.8f;
+#endif
+
+  const unsigned int MAX_BLOCKS = 80;
+  const unsigned int MAX_CONNECTED_MODULES = 40;
+
+  const unsigned int N_MAX_PIXEL_SEGMENTS_PER_MODULE = 50000;
+
+  const unsigned int N_MAX_PIXEL_MD_PER_MODULES = 2 * N_MAX_PIXEL_SEGMENTS_PER_MODULE;
+
+  const unsigned int N_MAX_PIXEL_TRIPLETS = 5000;
+  const unsigned int N_MAX_PIXEL_QUINTUPLETS = 15000;
+
+  const unsigned int N_MAX_PIXEL_TRACK_CANDIDATES = 30000;
+  const unsigned int N_MAX_NONPIXEL_TRACK_CANDIDATES = 1000;
+
+  const unsigned int size_superbins = 45000;
+
+  //defining the constant host device variables right up here
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniMulsPtScaleBarrel[6] = {0.0052, 0.0038, 0.0034, 0.0034, 0.0032, 0.0034};
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniMulsPtScaleEndcap[5] = {0.006, 0.006, 0.006, 0.006, 0.006};
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniRminMeanBarrel[6] = {
+      25.007152356, 37.2186993757, 52.3104270826, 68.6658656666, 85.9770373007, 108.301772384};
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float miniRminMeanEndcap[5] = {
+      130.992832231, 154.813883559, 185.352604327, 221.635123002, 265.022076742};
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float k2Rinv1GeVf = (2.99792458e-3 * 3.8) / 2;
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float kR1GeVf = 1. / (2.99792458e-3 * 3.8);
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float sinAlphaMax = 0.95;
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float ptCut = PT_CUT;
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float deltaZLum = 15.0;
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float pixelPSZpitch = 0.15;
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float strip2SZpitch = 5.0;
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float pt_betaMax = 7.0;
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float magnetic_field = 3.8112;
+  // Since C++ can't represent infinity, SDL_INF = 123456789 was used to represent infinity in the data table
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float SDL_INF = 123456789.0;
+}  //namespace SDL
+
+namespace T5DNN {
+  // Working points matching LST fake rate (43.9%) or signal acceptance (82.0%)
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float LSTWP1 = 0.3418833f;  // 94.0% TPR, 43.9% FPR
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float LSTWP2 = 0.6177366f;  // 82.0% TPR, 20.0% FPR
+  // Other working points
+  ALPAKA_STATIC_ACC_MEM_GLOBAL const float WP70 = 0.7776195f;    // 70.0% TPR, 10.0% FPR
```

Comment:
Comment from Matt:
> are these all used somewhere? I found where the LSTWP are used, but not the other WPs, but I'm not sure I did an exhaustive search. In any case, could be a possibility for cleanup, if not used. These working points are not something you would want to make configurable?

What were our plans with all of these working points?

As for the reply, I would say that we will revisit them soon, so any changes to them (including making them configurable), could be deferred to a later time.